CacheMoney/frontend/src/config.js

```diff
@@ -1,11 +0,0 @@
-const config = {
-    url: "http://cachemoney-env.eba-vtpiesgf.us-east-1.elasticbeanstalk.com/",
-    // local: "http://localhost:8080/"
-    // remote: "http://cachemoney-env.eba-vtpiesgf.us-east-1.elasticbeanstalk.com/"
-    
-    url_img: "http://revature-cachemoney.s3-website-us-east-1.amazonaws.com/"
-    // local: "http://localhost:3000/"
-    // remote: "http://revature-cachemoney.s3-website-us-east-1.amazonaws.com/"
-};
-
-export default config;
```

Comment:
This should probably be added to the gitignore, and a bit about it in STARTUP.md if we're going to delete the file

Reply:
Added to .gitignore